### ISSUE TITLE
LINCS pretrained model weights and configs

### ISSUE THREAD

**Issue by jasperhyp** [theislab/chemCPA#123]
Hi @MxMstrmn @siboehm !  Thanks for your insightful work and sharing the code.  Is it possible for you to also share the configs and weights for models that are only pretrained on LINCS (before any finetuning)?  I am interested in applying those directly to another bulk perturbation dataset.  

If I understand correctly, they should be related to this section in the `finetuning_num_genes.json`:
```
'pretrained_model_hashes': {'grover_base': 'ff420aea264fca7668ecb147f60762a1',
   'jtvae': 'a7060ac4e2c6154e64a13acd414cbba2',
   'rdkit': '4f061dbfc7af05cf84f06a724b0c8563'}
```

~~In addition, is it possible for you to provide some guidelines as to how to pretrain the model with the losses you specified in `ComPert`, and also how to finetune pretrained models?~~
I found some training guidelines in `chemCPA/experiments_run.py` and `manual_run.py` -- please correct me if there's another place I should refer to!  It seems it's feasible to train without seml -- will try it out.

Thank you very much in advance.

**Comment by MxMstrmn** [comment]
Hi @jasperhyp, 

I will provide further training instructions for the LINCS data only soon. For now, you should be able to train chemCPA on L1000 by using the provided LINCS dataset and adjusting the `manual_run.yaml`.

**Comment by MxMstrmn** [comment]
Hi @jasperhyp, as discussed via email the pertained weights might not be what you need. In the meantime, can you check #138 - the preprocessing should now work and the `manual_run.py` should execute without issues and train on the LINCS data. It still uses semi but does not require a mongo DB in the background.

**Comment by jasperhyp** [comment]
Hi @MxMstrmn !  Thank you so much for the prompt updates!  I will try training the model today and will get back to you if I encounter any issues.

**Comment by jasperhyp** [comment]
Hi @MxMstrmn !  Sorry for the late response, could you also point me to the yaml config that you are using for LINCS after the sweep?  Thanks!

**Comment by MxMstrmn** [comment]
True, if it runs nicely with the `manual_run.yaml` that is already great. To mimic what we had for the LINCS training, please check `experiments/lincs_rdkit_hparam` folder which is what we did for the LINCS data.

**Comment by cbirchsy** [comment]
Hi @MxMstrmn, thanks for the updates. I am also interested in re-running the training on just LINCS. For a reference when trying to reproduce the pre-training, do you report any val r^2 metrics for the pre-training? 

I would also just be generally curious to see how the r^2 on LINCS compares to sci-plex

**Comment by cbirchsy** [comment]
> @cbirchsy the results are listed in Table 5 in the [paper](https://arxiv.org/pdf/2204.13545.pdf).

Thanks @MxMstrmn. Are the metrics shown in Table 5 reported for held-out drugs or held-out drug-covariate combinations?

**Comment by MxMstrmn** [comment]
This was a random split as we were not analysing the old prediction ability on the LINCS data. Hence, it is neither true ood drugs or drug-covariate combinations. 


Here, random really meant random as in a naive ML data split. 
> Table 5: Summary of chemCPA on the L1000 dataset for different molecule encoders G. All models were trained on the same random split. Reported are the overall disentanglement scores (drug and cell line) and the r2-scores on the test set.
